pkg/oauth2/tokenstore/mysqltokenstore/token_store.go

```diff
@@ -19,12 +19,15 @@ import (
 	"encoding/json"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/jinzhu/gorm"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/types/known/timestamppb"
 	"gopkg.in/oauth2.v3"
 	"gopkg.in/oauth2.v3/models"
 
+	"github.com/erda-project/erda-proto-go/core/token/pb"
```

Comment:
it's wired the `pkg/oauth2/tokenstore/mysqltokenstore` depend on `erda-proto-go`, maybe this pkg should not exist and all code move to `core-services` as plain CRUD.

Reply:
What do you think? @sfwn

Reply:
Add TODO, will create a new pr for it.